invite/src/invite_bot.ts

```diff
@@ -51,10 +51,16 @@ export class InviteBot {
    * event that the invite fails to send for some reason. If left as null, the
    * comment will just say "ask someone for help".
    * Example: 'ampproject/wg-infra'
+   *
+   * The allowTeamSlug parameter restricts the set of users who can trigger an
+   * invite to members of a GitHub team. The value should be the name of the 
+   * team, but should not include the org.
+   * Example: 'wg-infra'
    */
   constructor(
     client: Octokit,
     private org: string,
+    private allowTeamSlug: string,
```

Comment:
Might be worth making this an array or strings, I can 100% see this becoming a feature request ("oh can you also let the TSC and AC invite people?")

Reply:
If they do that, I'll push back and ask them to create a Team which includes subteams (nesting is allowed). More sensible than doing multiple API checks, and easier to reason about/keep track of a single `ampproject/inviters` team

Reply:
Teams _aren't_ cheap, their listings require constant maintenance - this pushback means you expect Joey to keep adding people to multiple teams all the time. I'll accept the pushback, but urge you to reconsider

Reply:
No, I mean groups can own groups. So if he wants `tsc-meta` and `wg-infra` and `wg-approvers` to all have invite powers, he can add the _groups themselves_ to something like `org-inviters`. Then when someone gets added to `wg-infra`, that automatically includes them in `org-inviters`. Do you get what I mean with that?

Reply:
Yep :) alright